packages/flutter_tools/lib/src/commands/build_windows.dart

```diff
@@ -65,6 +67,16 @@ class BuildWindowsCommand extends BuildSubCommand {
         flutterUsage: globals.flutterUsage,
       ),
     );
+
+    // We don't print a size because the containing directory may contain
+    // additional files not seen by the user and because the binary is not
+    // self-contained.
+    globals.logger.printStatus(
+      '${globals.logger.terminal.successMark}  '
```

Comment:
Is the double space intentional here? It looks like other platforms don't have that.

```suggestion
      '${globals.logger.terminal.successMark} '
```